src/facets/BatchExecutionFacet.sol

```diff
@@ -79,6 +79,47 @@ contract NiftyApesBatchExecutionFacet is INiftyApesStructs, INiftyApesErrors, IB
         }
     }
 
+    /// @inheritdoc IBatchExecution
+    function borrowBatch(
+        Offer[] calldata offers,
+        bytes[] calldata signatures,
+        address borrower,
+        uint256[] calldata tokenIds,
+        uint256[] calldata tokenAmounts,
+        bool partialExecution
+    ) external returns (uint256[] memory loanIds) {
+        // requireLengthOfAllInputArraysAreEqual
+        if (
+            offers.length != signatures.length ||
+            offers.length != tokenIds.length ||
+            offers.length != tokenAmounts.length
+        ) {
+            revert InvalidInputLength();
+        }
+        loanIds = new uint256[](offers.length);
+        // loop through list of offers to execute
+        for (uint256 i; i < offers.length; ++i) {
+            // instantiate calldata params to bypass stack too deep error
+            Offer memory offer = offers[i];
+            // bytes calldata signature = signatures[i];
```

Comment:
Do we need these commented out variables?

Reply:
nope, removed.